### PR TITLE
feat(darkTheme): set up darkTheme for SelectInputLabelBox

### PR DESCRIPTION
**Pull request by ponddap**

### CODE REVIEW

File: src/components/SelectInputLabelBox/SelectInputLabelBox.js [hunk 1]
```diff
@@ -63,10 +68,15 @@ export const Value = styled.button`
   text-align: left;
   ${typography.subhead1};
   color: ${(props) => {
+    console.log(props);
```
Comment (seanevd) [inline src/components/SelectInputLabelBox/SelectInputLabelBox.js]: left a console.log

**Comment by ponddap** [comment]
console.log removed

File: src/components/SelectInputLabelBox/SelectInputLabelBox.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ import PropTypes from 'prop-types';
 const padding = '16px';
 
 export const Label = styled.div`
-  color: ${colors.black40};
+  color: ${props => props.theme.labelColor || colors.black40};
```
Comment (cozzbp) [inline src/components/SelectInputLabelBox/SelectInputLabelBox.js]: Would it make sense to just add the default theme to defaultProps?

Comment (gabe-lei) [reply src/components/SelectInputLabelBox/SelectInputLabelBox.js]: Just want to confirm if theme has a default value of {}, so it won't typeError here.

**Pull request merged**